book/templates/case-study-template/README.md

```diff
@@ -0,0 +1,88 @@
+# Welcome
+
+Thank you for deciding to write a case study for _The Turing Way_ book!
+
+To standardise the book's content and maintain consistency throughout, we have created this template to guide you as you write your case study.
+
+## Before You Begin
+
+_The Turing Way_ is made up of Guides which span several chapters.
+For every Guide, there is a `Case Studies` chapter that contains a collection of case studies that explore how key concepts from the Guide relate to specific subjects (such as a project, person, organisation, or phenomenon).
+
+Structurally, a Guide in _The Turing Way_ Github repository is a folder and your case study is a file in the `case-studies` sub-folder within it.
+All Guides in _The Turing Way_ live in the `book/website` directory.
+
+For example, if you wrote the case study about the [Statistical Methods Manuscript](https://the-turing-way.netlify.app/reproducible-research/case-studies/statistical-methods-manuscript.html) in the Guide for Reproducible Research, then the file structure could resemble the following:
+
+```
+book/website
+│
+└───reproducible-research <---- (Folder for the Guide to Reproducible Research)
+│   │   reproducible-research.md <---- (Guide's landing page)
+|   |   vcs.md <---- (Landing page for the Version Control chapter)
+│   │   case-studies.md <---- (Landing page for Case Studies chapter)
+│   │
+│   └───case-studies
+│       │   statistical-methods-manuscript.md <---- (Statistical Manuscripts Case Study)
+|       |   Case Study 2.md
+|       |   ...
+│   
+└───project-design <---- (Folder for the Guide for Project Design)
+    │   project-design.md
+    │   ...
+```
```

Comment:
```suggestion

```
book/website
│
└───reproducible-research <---- (Folder for the Guide to Reproducible Research)
│   └───binderhub
|   └─── case-studies  <---- (folder containing subchapter for case studies the Guide to Reproducible Research)
│       │   statistical-methods-manuscript.md <---- (Statistical Manuscripts Case Study)
|       |   example-case-study-file.md
|   |   ...
│   │   case-studies.md <---- (Landing page for Case Studies chapter)
|   |   ...
│   
└───project-design <---- (Folder for the Guide for Project Design)
    │   project-design.md
    │   ...
```
```